bin/nrnivmodl.in

```diff
@@ -19,18 +19,26 @@ else
 fi
 
 if test "${NRNHOME_EXEC}" != "" ; then
-	exec_prefix="${NRNHOME_EXEC}"
```

Comment:
The entire ```if test``` no longer makes sense to me. Why would a non-empty NRNHOME_EXEC cause  bindir and libdir to be derived from exec_prefix and not NRNHOME_EXEC. Maybe a comment would help.

Reply:
I also find hard to grasp, but that logic was not changed.

Reply:
It seems to me the logic HAS changed. The master branch says:
```
if test "${NRNHOME_EXEC}" != "" ; then
        exec_prefix="${NRNHOME_EXEC}"
        bindir="${exec_prefix}/bin"
        libdir="${exec_prefix}/lib"
fi
```
So if NRNHOME_EXEC has a value, then it gets put in exec_prefix, bindir, and libdir.